src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -973,6 +989,20 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
     std::stringstream run_details;
     size_t block_size = 16 * 1024 * 1024; // Default block size 16MB
 
+    //check custom argument from user
+    auto str_block_size = _ptTest.get<std::string>("block-size", "");
+    if (!str_block_size.empty()) {
+      try {
+        block_size = static_cast<size_t>(std::stoll(str_block_size, nullptr, 0));
+      }
+      catch(const std::invalid_argument&) {
+        std::cerr << boost::format("ERROR: '%s' is an invalid argument. Please specify block-size in bytes like 'p2p::block-size:1024'\n") % str_block_size;
```

Comment:
```suggestion
        std::cerr << boost::format("ERROR: The parameter '%s' value '%s' is invalid for the test '%s'. Please specify and integer byte block-size.'\n") % "block-size" % str_block_size % "dma" ;
```
Note: It would be create if the parameter and test name was data driven.